README.md

```diff
@@ -2,3 +2,225 @@
 <img src="./Images/Logo.svg" alt="Level Logo" width="200" height="100" align="left">
 A 2D chat with end-to-end encryption and markdown support
 
+# Installation
+
+## Flutter
+
+Based on https://docs.flutter.dev/get-started/install/linux/android
+
+Required packages:
+```bash
+sudo apt-get update -y && sudo apt-get upgrade -y;
+sudo apt-get install -y curl git unzip xz-utils zip libglu1-mesa
+```
+
+IDE Extensions: [VSCode](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter), [emacs](https://jwill.dev/blog/2022/01/24/EmacsAsIDE-Flutter.html)
+
+Disable telemetrics for VSCode extension:
+```bash
+dart --disable-analytics
```

Comment:
observation: 
Der Befehl 'dart' wurde nicht gefunden, kann aber installiert werden mit:

sudo snap install flutter

also [validation](https://docs.flutter.dev/get-started/editor?tab=vscode) of VS Code flutter extension unsuccessful.